study-builder/studies/circadia/i18n/en.json

```diff
@@ -519,5 +519,67 @@
 
   "pcq_name": "Post Collection Questionnaire",
   "pcq_title": "Post Collection Questionnaire",
-  "pcq_header": "Genetic Info Release Request"
+  "pcq_header": "Genetic Info Release Request",
+
+  "pcq_bedtime_sleep_earlier_or_later_question": "For your bedtime during the study, did you ever go to sleep earlier/later than the time you entered on the Mini Sleep Log for that night? ",
+  "pcq_bedtime_sleep_earlier_yes": "Yes (I went to sleep earlier)",
+  "pqc_bedtime_sleep_later_yes": "Yes (I went to sleep later)",
+  "pcq_bedtime_sleep_earlier_no": "No",
+  "pcq_bedtime_sleep_earlier_text_field": "If yes, please provide details that you best remember",
+
+  "pcq_waketime_earlier_or_later_question": "For your waketime during the study, did you ever wake up earlier/later than the time you entered on the Mini Sleep Log for that day?",
+  "pcq_waketime_earlier_yes": "Yes (I woke up earlier)",
+  "pcq_waketime_earlier_no": "No (I woke up later)",
+  "pcq_waketime_no": "No",
+  "pcq_waketime_earlier_or_later_text_field": "If yes, please provide details that you best remember",
+
+  "pcq_activity_tracker_off_question": "Did you ever take your activity tracker off and not report it?",
+  "pcq_activity_tracker_yes": "Yes",
+  "pcq_activity_tracker_no": "No",
+
+  "pqc_activity_tracker_problems_question": "Did you have any problems wearing the activity tracker?",
+  "pcq_activity_tracker_problems_yes": "Yes",
```

Comment:
No need to have different variables for the same string. I recommend to have one (for example, "answer_yes").